docs/user/howto_test_offline.md

```diff
@@ -27,7 +33,8 @@ operation.
 /usr/bin/microshift-cleanup-data --ovn
 ```
 
-3. Determine the interface on the host that has the default route
+3. Determine the interface on the host that has the default route.
+   Make sure you only have one default route configured
```

Comment:
I think it just takes the first one if there are multiples, so maybe we should adjust for that by making the command on line 40 something like 

```
IFACE=$(ip route | head -n 1 | grep default | awk '{print $5}')
```